### DISCUSSION TITLE
Failed to load

### DISCUSSION THREAD

**Discussion by DavinHsu** [vslinko/obsidian-zoom#41] [Archive]
![failed-to-loadPlugin](https://user-images.githubusercontent.com/22023184/146643190-5f661ac9-5e5b-4ffe-8d38-e3d1ded4d0a8.png)
windows10+obsidianV0.12.19
why Failed to load when it to be enabled ?

**Comment by vslinko** [comment]
It looks like you somehow got the newest version, which does not support Obsidian 0.12.
I don't know how this happened. It seems to be a bug in Obsidian version checker.

I could offer you two options:
- you can wait for Obsidian 0.13 to be publicly released
- you can install https://github.com/vslinko/obsidian-zoom/releases/tag/0.1.2 manually by copying files to `.obsidian/plugins/obsidian-zoom`

**Reply by DavinHsu** [reply]
But, This error was originally caused by my manual installation.

**Reply by vslinko** [reply]
This is good. It means that there is no problem with Obsidian version checking.

So the problem is that obsidian-zoom 0.2 is not compatible with Obsidian 0.12. Please install obsidian-zoom 0.1.2 or wait for the public release of Obsidian 0.13.
